db/dcrsqlite/apisource.go

```diff
@@ -1045,9 +1045,22 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 		return nil
 	}
 
+	maxcount := explorer.MaxAddressRows
 	txs, err := db.client.SearchRawTransactionsVerbose(addr,
 		int(offset), int(count), true, true, nil)
-	if err != nil {
+	maxLimitOfTxs, err := db.client.SearchRawTransactionsVerbose(addr,
```

Comment:
I'm not OK with calling this RPC twice.  Looks like this is only being used to get the count `int64(len(maxLimitOfTxs))`, and `maxLimitOfTxs` is otherwise not used.

We have a hard limit (`maxcount`), so this is not even getting the real total count.

Think about if `len(txs)` were less than `count`... the second RPC is pointless.

Also, you could get by with just the second RPC with the larger `maxcount`, and then only use `count` of the transactions to build the `[]*explorer.AddressTx` with `makeExplorerAddressTx` below.